rest-api/tools/import_organizations.py

```diff
@@ -48,6 +48,7 @@
 SITE_NOTES_COLUMN = 'Notes'
 # TODO: switch this back to 'Status' [DA-538]
 SITE_STATUS_COLUMN = 'PTSC Scheduling Status'
+ENROLLING_STATUS_COLUMN = 'Actively Enrolling Status'
```

Comment:
Let's ask Ashley what she wants to call this.

Reply:
I've reached out, still waiting on response.

Reply:
OK. For now, just call this "Status" to match what's in the sheet already?

Reply:
done